files/mcollective/pluginpackager/templates/aiomodule/README.md.erb

```diff
@@ -10,7 +10,8 @@
 
 <%= @plugin.metadata[:description] %>
 
-The <%= module_name %> module is based on the source from <%= @plugin.metadata[:url] %>.
+The <%= module_name %> module generated automatically, based on the source from <%= @plugin.metadata[:url] %>.
+You can find templates used for generation in https://github.com/choria-io/puppet-mcollective/tree/master/files/mcollective/pluginpackager/templates/aiomodule.
```

Comment:
The point of this `mco plugin package` is that any user of mcollective can package their modules, not just Puppet labs ones, any and all users.

Using it to produce whats on the choria forge means you get *exactly* the same quality modules for your own use as the project does, users are first class citizens not somehow left out when $vendor decides to stop caring or invent tooling that they refuse to share with users, as has historically happened with these.

So linking to puppetlabs there wouldn't be useful.

How about:

```
Development:

To contribute to this MCollective plugin please visit <%= @plugin.metadata[:url] %>.  This module was generated using the Choria Plugin Packager based on templates found at the [GitHub Project](https://github.com/choria-io/puppet-mcollective).
```

I don't want to link to `puppet-mcollective` because its going to change and these modules will be long lived, am not going to make a bunch of dead links knowlingly :)